tmt/steps/execute/__init__.py

```diff
@@ -168,9 +168,14 @@ def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecor
                 help='Stop execution after the first test failure.'),
             ] + super().options(how)
 
-    def go(self, guest: Guest) -> None:
-        super().go(guest)
-        self.verbose(
+    def go(
+            self,
+            *,
```

Comment:
I am a noob, when other noob's read this code, they should be given a hint here:

```suggestion
            *,  # NOTE: accept only keyword arguments, https://peps.python.org/pep-3102/
```

Reply:
I am not sure how else to help the user. But I had to really google this myself, and it was not super obvious to google out, or why we want it this way

Reply:
if this is just me, feel free to ignore

Reply:
I believe it won't be that hard after some time :) It's used already by base classes (everything derived from `Common`), and while it's not as new as the walrus operator (`:=`), it's a valid Python 3 construct. After some getting used to you shouldn't consider it that exceptional :)

Reply:
ok, it was just me noob :)